### ISSUE TITLE
Difference accessing methods on compiled vs uncompiled player?

### ISSUE THREAD

**Issue by fredrik-telia** [shaka-project/shaka-player#7416]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
4.11.7

**What browser and OS are you using?**
Mac/Chrome

**Please ask your question**
Ok, maybe I'm misunderstanding something here, I'm trying to access the new `getFetchedPlaybackInfo()`-method, using version 4.11.7, but I can't.

I decided to check the demo app, and using the compiled (release) version I get:
```Uncaught TypeError: p.getFetchedPlaybackInfo is not a function```
when running the following in the console:
```
p = new shaka.Player()
p.getFetchedPlaybackInfo()
```

However, using either the debug version or the uncompiled version, it works as expected:
```
{video: null, audio: null, text: null}
```

tl;dr:
using the debug or uncompiled version of the demo player let's me create a new shaka player and access getFetchedPlaybackInfo(), but using the compiled version I cannot.

**Comment by Iragne** [comment]
Hello if i'm not wrong this function is not export to the public interface. that is why you can have it in debug and not in compile version 
we should have * @export on it
or maybe i miss also something

**Comment by joeyparrish** [comment]
This was added in https://github.com/shaka-project/shaka-player/pull/7074.  Based on the details in that PR (externs defined for the return value, added to method list in a cast test), I think it was intended to be exported.  Adding the `@export` annotation should fix it.

**Comment by Iragne** [comment]
I just push the PR ;)

**Comment by fredrik-telia** [comment]
> I just push the PR ;) @fredrik-telia

Excellent, thank you @Iragne !
